### PR TITLE
Convert Batch Management lib to .NET Core

### PR DESCRIPTION
**Pull request by jasper-schneider**
Converted the Batch Management lib and test project to .NET Core.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jasper-schneider__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaschnei). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -20,8 +20,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Batch Management Library")]
 [assembly: AssemblyDescription("Provides management functions for Microsoft Azure Batch services.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```
Comment (markcowl) [inline src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/Properties/AssemblyInfo.cs]: normally, you want AssemblyVersion to just have the the major version and the fileversion to match the package version

**Comment by markcowl** [comment]
@jasper-schneider one nit, otherwise lgtm

File: src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (jasper-schneider) [inline src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/Properties/AssemblyInfo.cs]: Got it, updated

**Pull request merged**